src/state_machine/mod.rs

```diff
@@ -22,12 +22,18 @@ pub enum DkgError {
     /// DKG public timeout
     #[error("DKG public timeout, waiting for {0:?}")]
     DkgPublicTimeout(Vec<u32>),
+    /// DKG private timeout
+    #[error("DKG private timeout, waiting for {0:?}")]
+    DkgPrivateTimeout(Vec<u32>),
     /// DKG end timeout
     #[error("DKG end timeout, waiting for {0:?}")]
     DkgEndTimeout(Vec<u32>),
     /// DKG crypto error
     #[error("DKG crypto error")]
     Crypto(#[from] DkgCryptoError),
+    /// Bad state for DKG
+    #[error("Bad state for DK")]
```

Comment:
Typo. Should be "DKG"

Reply:
I think I can remove this entirely, IIRC I didn't end up using it since the DkgStatus::Failure just takes a string?

Reply:
Removed.